src/pages/crm-reviews-page/ReviewsList/ReviewsListItem/ReviewsListItem.tsx

```diff
@@ -0,0 +1,128 @@
+import styles from './reviews-list-item.module.scss'
+import IconPlus from '../../../../icons/plus.svg?react'
+import { ReviewsRating } from '../ReviewsRating/ReviewsRating'
+import IconButton from '@mui/material/IconButton'
+import DeleteIcon from '../../../../icons/delete.svg?react'
+import Tooltip from '@mui/material/Tooltip'
+import ChangeIcon from '../../../../icons/crm-file.svg?react'
+import { Images } from '../../../../redux/feedbacks/slice'
+import { ModalCustom } from '../../../../components/Modal-custom-btn/ModalCustomWindow'
+import { useState } from 'react'
+import axiosInstance from '../../../../axios/settings'
+
+export interface ReviewsListItemProps {
```

Comment:
same here

Reply:
and check be types, should be already created. use types from be and do not create your own